tasks/oah-update.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Checking latest version
+  shell: 'curl -s https://api.github.com/repos/opensbom-generator/spdx-sbom-generator/releases/latest | grep "browser_download_url.*linux-amd64.tar.gz" | cut -d : -f 2,3 | tr -d \" | head -1 | xargs'
+  register: update_link
+  warn: false
+  
+- name: Checking if latest version is installed
+  debug:
+    msg: "Latest version already installed"
+  when: "'{{ update_link.stdout }}' == '{{ release_link }}/{{ latest_version }}'"

```

Comment:
Syntax incorrect.

![image](https://user-images.githubusercontent.com/58501088/190560561-8e87a78d-1b56-4409-b9be-6921dc1ba02c.png)

Reply:
Changed the when condition which was creating this Syntax error. Now working fine.